src/core/ext/transport/inproc/inproc_transport.cc

```diff
@@ -186,6 +185,7 @@ struct inproc_stream {
       if (cs->write_buffer_cancel_error != GRPC_ERROR_NONE) {
         cancel_other_error = cs->write_buffer_cancel_error;
         cs->write_buffer_cancel_error = GRPC_ERROR_NONE;
+        maybe_process_ops_locked(this, cancel_other_error);
```

Comment:
why is this needed now, but not earlier? Is it just early execution unrelated to the fix?

Reply:
Somehow it tripped now but never before (or maybe it tripped but we never noticed it since there were other bugs too). The issue was that if we buffered a cancellation, we never actually processed that operation after unbuffering it.

Reply:
In all likelihood, we didn't see it before because closure scheduling changed the time at which the effect happened.